perf/lua/benchmark.lua

```diff
@@ -119,13 +123,37 @@ function M.argparse(arg, argtable, custom_help)
     return params
 end
 
+local function load_average()
+    local path = '/proc/loadavg'
+    local fh = assert(fio.open(path, {'O_RDONLY'}))
+    local loadavg_buf = fh:read(1024)
+    fh:close()
+    -- Format is '0.89 0.66 0.80 2/1576 1203510'.
+    local loadavg_re = '(%d+.%d+) (%d+.%d+) (%d+.%d+)'
+
+    return { loadavg_buf:gmatch(loadavg_re)() }
```

Comment:
Why do we need the additional variable `loadavg_re`? Also, I don't understand why do we use `gmatch()` instead of `match()`?